### PR TITLE
netutils: add support to save wapi config to file system

### PR DESCRIPTION
**Pull request by anchao**
Change-Id: I4f7e9a8e4b4f38f08bd0a1ede1994e24f80bf89b
Signed-off-by: chao.an <anchao@xiaomi.com>

### CODE REVIEW

File: include/wireless/wapi.h [hunk 1]
```diff
@@ -95,6 +95,12 @@
 # define SOCK_WAPI SOCK_STREAM
 #endif
 
+#ifndef CONFIG_WIRELESS_WAPI_INITCONF
+#define wapi_load_config(ifname, confname, conf) NULL
```
Comment (Ouss4) [inline include/wireless/wapi.h]:
```suggestion
#  define wapi_load_config(ifname, confname, conf) NULL
```

File: include/wireless/wapi.h [hunk 2]
```diff
@@ -95,6 +95,12 @@
...
+#define wapi_unload_config(load)
```
Comment (Ouss4) [inline include/wireless/wapi.h]:
```suggestion
#  define wapi_unload_config(load)
```

File: include/wireless/wapi.h [hunk 3]
```diff
@@ -95,6 +95,12 @@
...
+#define wapi_save_config(ifname, confname, conf) 0
```
Comment (Ouss4) [inline include/wireless/wapi.h]:
```suggestion
#  define wapi_save_config(ifname, confname, conf) 0
```

File: include/wireless/wapi.h (hunk 1 above)
Comment (anchao) [reply include/wireless/wapi.h]: Done.

File: include/wireless/wapi.h (hunk 2 above)
Comment (anchao) [reply include/wireless/wapi.h]: Done.

File: include/wireless/wapi.h (hunk 3 above)
Comment (anchao) [reply include/wireless/wapi.h]: Done.

**Pull request merged**